test/sharness/t0080-repo.sh

```diff
@@ -74,6 +74,16 @@ test_expect_success "file no longer pinned" '
 	test_sort_cmp expected2 actual2
 '
 
+test_expect_success "recursively pin afile(default action)" '
+	HASH=`ipfs add -q afile` &&
```

Comment:
you can reuse the `HASH` variable from the previous statement, no need to call add every time. `sh` is globally scoped and kindof confusing sometimes...